game_eggs/steamcmd_servers/space_engineers/torch/README.md

```diff
@@ -0,0 +1,38 @@
+# Space Engineers Torch Server
+
+### Author & Contributers
+| Name        | Github Profile  |
+| ------------- |-------------|
+|   gOOvER   | https://github.com/gOOvER |
+
+
+## Description
+Torch is the successor to SE Server Extender and gives server admins the tools they need to keep their Space Engineers servers running smoothly. It features a user interface with live management tools and a plugin system so you can run your server exactly how you'd like. Torch is still in early development so there may be bugs and incomplete features.
+
+**NOTE**
+Because of the winetricks installation, the install process could take some time. Likewise, the first start of the server takes its time.
+
+When installed, the Server loads a base World and config from this repo. New world must be generated with the Windows Dedicated Server and then uploaded to the /config/Saves folder.
+Everything which is not defined in a variable could be edited directy in the config file /config/SpaceEngineers-Dedicated.cfg and config/Saves/{{WORLD}}/Sandbox.sbc
+
+## Server Ports
+
+Default server ports are listed below, but the Main, STEAm and API port can be any port. 
```

Comment:
```suggestion
Default server ports are listed below, but the Main, Steam and API port can be any port. 
```

Reply:
only Copy and Paste from their site

Reply:
The idea behind the change from "Main" to "Game" was due to the table with the required ports. There they are called "Game, Steam and API"